docs/deployment_guide/partner_editable/post_deployment.adoc

```diff
@@ -31,12 +31,17 @@
 |`TanzuApiToken`
 |API Token
 |
-|{partner-registry} UAA API refresh token. For more information, refer to https://network.tanzu.vmware.com/docs/api[{partner-registry} API documentation^].
+|VMware Tanzu Network user account and authentication (UAA) API refresh token. For more information, refer to https://network.tanzu.vmware.com/docs/api[VMware Tanzu Network API Documentation].
```

Comment:
Please keep the settings as is {partner-registry} - This is a General comment across all files.